generator/pom.xml

```diff
@@ -87,10 +88,7 @@
                             <sourcePaths>
                                 <sourcePath>${parent.project.dir}/spec/schemas/artifactpackaged.json</sourcePath>
                             </sourcePaths>
-                            <outputDirectory>${sdk.project.dir}/src/main/java</outputDirectory>
                             <targetPackage>dev.cdevents.models.artifact.packaged</targetPackage>
-                            <includeHashcodeAndEquals>true</includeHashcodeAndEquals>
-                            <includeToString>false</includeToString>
```

Comment:
How come these are going away, but the hash and equals method are still there? I guess it's the default behaviour to generate those?

Reply:
These settings are inherited from the common plugin configuration. Defining them on every execution was redundant.